easypost/shipment.py

```diff
@@ -19,11 +20,27 @@
 
 
 class Shipment(AllResource, CreateResource):
-    def regenerate_rates(self) -> "Shipment":
+    @classmethod
+    def create(cls, api_key: Optional[str] = None, carbon_offset: bool = False, **params) -> Any:
```

Comment:
This does not return anything, it returns a Shipment object.

```suggestion
    def create(cls, api_key: Optional[str] = None, carbon_offset: bool = False, **params) -> "Shipment":
```